PhysicsTools/PatAlgos/python/slimming/customizeMiniAOD_MuEGFixMoriond2017.py

```diff
@@ -0,0 +1,138 @@
+    process.patMuons.userData.userInts.src = [ cms.InputTag("muonsCleaned:oldPF") ]
+
+    process.load("CommonTools.ParticleFlow.pfCandidatesBadMuonsCleaned_cfi")
+    process.muonsCleaned.badmuons = badMuons
+    replaceMuons = MassSearchReplaceAnyInputTagVisitor("muons", "muonsCleaned", verbose=verbose)
+    needOriginalMuons = [ process.muonsCleaned ] + [ getattr(process,l.moduleLabel) for l in badMuons ]
+    replacePFCandidates = MassSearchReplaceAnyInputTagVisitor("particleFlow", "pfCandidatesBadMuonsCleaned", verbose=verbose)
+    for everywhere in [ process.producers, process.filters, process.analyzers, process.psets, process.vpsets ]:
+        for name,obj in everywhere.iteritems():
+            if obj not in needOriginalMuons:
+                replaceMuons.doIt(obj, name)
+            if obj != process.pfCandidatesBadMuonsCleaned: 
+                replacePFCandidates.doIt(obj, name)
+            
+    process.load("CommonTools.ParticleFlow.pfEGammaToCandidateRemapper_cfi")
+    process.pfEGammaToCandidateRemapper.pf2pf = cms.InputTag("pfCandidatesBadMuonsCleaned")
+    process.reducedEgamma.gsfElectronsPFValMap = cms.InputTag("pfEGammaToCandidateRemapper","electrons")
+    process.reducedEgamma.photonsPFValMap      = cms.InputTag("pfEGammaToCandidateRemapper","photons")
+    if hasattr(process,"gedGsfElectronsGSFixed"):
+        # also reconfigure pfEGammaToCandidateRemapper because of GS Fix
+        # first the old one
+        process.pfEGammaToCandidateRemapperBeforeGSFix = process.pfEGammaToCandidateRemapper.clone()
+        process.reducedEgammaBeforeGSFix.gsfElectronsPFValMap = cms.InputTag("pfEGammaToCandidateRemapperBeforeGSFix","electrons")
+        process.reducedEgammaBeforeGSFix.photonsPFValMap      = cms.InputTag("pfEGammaToCandidateRemapperBeforeGSFix","photons")
+        # then the new one
+        process.pfEGammaToCandidateRemapper.electrons = cms.InputTag("gedGsfElectronsGSFixed")
+        process.pfEGammaToCandidateRemapper.photons   = cms.InputTag("gedPhotonsGSFixed")
+        process.pfEGammaToCandidateRemapper.electron2pf = cms.InputTag("particleBasedIsolationGSFixed","gedGsfElectrons")
+        process.pfEGammaToCandidateRemapper.photon2pf   = cms.InputTag("particleBasedIsolationGSFixed","gedPhotons")
+
+
+def addDiscardedPFCandidates(process, inputCollection, verbose=False):
+    process.primaryVertexAssociationDiscardedCandidates = process.primaryVertexAssociation.clone(
+        particles = inputCollection,
+        )
+    process.packedPFCandidatesDiscarded = process.packedPFCandidates.clone(
+        inputCollection = inputCollection,
+        PuppiNoLepSrc = cms.InputTag(""),
+        PuppiSrc = cms.InputTag(""),
+        secondaryVerticesForWhiteList = cms.VInputTag(),
+        vertexAssociator = cms.InputTag("primaryVertexAssociationDiscardedCandidates","original")
+        )
+    addKeepStatement(process, "keep patPackedCandidates_packedPFCandidates_*_*",
```

Comment:
```addKeepStatement ``` is not in this PR.
Please include a port of this method definition as well or drop these customizations from this PR as non-functional.